src/services/litegraphService.ts

```diff
@@ -152,7 +153,10 @@ export const useLitegraphService = () => {
           })
         }
 
-        if (PRIMITIVE_TYPES.has(inputSpec.type)) {
+        if (
+          PRIMITIVE_TYPES.has(inputSpec.type) ||
```

Comment:
nit: We should probably remove `PRIMITIVE_TYPES.has(inputSpec.type)` check. They should also be opt-out-able.